src/comment/comment.routes.ts

```diff
@@ -0,0 +1,10 @@
+import { Router } from "express";
+import { isAuthenticated } from "../middleware/authMiddleware";
+import { CommentController } from "./comment.controller";
+
+const router = Router();
+
+router.post('/blogs/:id/comments', isAuthenticated, CommentController.addComment)
+router.get('/blogs/:id/comments', CommentController.getComments)
+router.put('/blogs/:id/comments/:commentId', isAuthenticated, CommentController.editComment)
```

Comment:
you have to add one check for edit and delete. Only comment owner can edit comment. For deleting comment owner and post owner can delete it

Reply:
Created and added isCommentAuthor and isCommentAuthOrAdmin middlewares